extensions/sql-migration/src/wizard/loginMigrationTargetSelectionPage.ts

```diff
@@ -37,12 +37,18 @@ export class LoginMigrationTargetSelectionPage extends MigrationWizardPage {
 	private _azureResourceDropdown!: azdata.DropDownComponent;
 	private _resourceSelectionContainer!: azdata.FlexContainer;
 	private _resourceAuthenticationContainer!: azdata.FlexContainer;
+	private _targetPortLabel!: azdata.TextComponent;
 	private _targetUserNameInputBox!: azdata.InputBoxComponent;
 	private _targetPasswordInputBox!: azdata.InputBoxComponent;
+	private _targetPortInputBox!: azdata.InputBoxComponent;
 	private _testConectionButton!: azdata.ButtonComponent;
 	private _connectionResultsInfoBox!: azdata.InfoBoxComponent;
 	private _migrationTargetPlatform!: MigrationTargetType;
 
+	private readonly _defaultSqlPort: string = "1433";
```

Comment:
should the default port for SQL MI be used here:  3342

Reply:
SQL MI by default has private endpoint enabled with port 1433.  3342 is the default port for public endpoint. So keeping 1433 as the default port.